js/src/Ice/.gitignore

```diff
@@ -19,5 +19,3 @@ RemoteLogger.d.ts
 RemoteLogger.js
 Router.d.ts
 Router.js
-RouterF.d.ts
-RouterF.js
```

Comment:
There's also some leftover references in this file as well:
- ProcessF.d.ts
- ProcessF.js
- LocatorF.d.ts
- LocatorF.js